docs/network-flow-visibility.md

```diff
@@ -691,16 +691,21 @@ Mouse out or click on the background will bring all the traffic back.
 
 ### Dashboard Customization
 
-If you would like to make any changes to any of the pre-built dashboards, or build
+If you would like to make any change to any of the pre-built dashboards, or build
 a new dashboard, please follow this [doc](https://grafana.com/docs/grafana/latest/dashboards/)
 on how to build a dashboard.
 
-By clicking on the "Save dashboard" button in the Grafana UI, the changes to the
-dashboards will be persisted in the Grafana database at runtime, but they will be
-lost after restarting the Grafana deployment. To restore those changes after a restart,
+From Theia 0.2, we use PersistentVolume for the Grafana configuration database.
+If you create a new dashboard or modify the pre-built dashboards, once you click
+on the "Save dashboard" button, the changes will be kept after a Grafana
+Deployment restart. Other changes in settings like passwords and preferences will
```

Comment:
Since grafana-pv is a hostPath volume I think we should clarify that setting are stored as long as the grafana pod is deployed on the same host. In order to make sure settings are preserved regardless of where the grafana pod is deployed, users should adjust grafana-pv configuration to use nfs or some other form of shared storage.

Reply:
Got it. Do you think we could use the similar PV solution as for ClickHouse, which allows user to choose a type of the PersistentVolume created by us or provide `persistentVolumeClaimSpec`? We can set `HostPath` as the default type and support Local and NFS options.

Reply:
I add a patch to support customized Grafana PV and update the doc. Please take a look if it is better.

Reply:
Great job!